src/api/nodeRpcCall.js

```diff
@@ -1733,13 +1719,13 @@ const getStakingBondingDuration = async () => {
   return api.consts.staking.bondingDuration;
 }
 
-const congressProposeLegislation = async (tier, index, legislationContent, walletAddress, callback) => {
+const congressProposeLegislation = async (tier, id, legislationContent, walletAddress, callback) => {
   const api = await getApi();
   const injector = await web3FromAddress(walletAddress);
 
   const threshold = await congressMajorityThreshold();
 
-  const proposal = api.tx.liberlandLegislation.addLaw(tier, index, legislationContent);
+  const proposal = api.tx.liberlandLegislation.addLaw(tier, id, legislationContent);
   const extrinsic = api.tx.council.propose(threshold, proposal, proposal.length);
```

Comment:
Can we also change "signAndSend" to "submitExtrinsic" ?

Reply:
Nah. This will at least double the size of this PR (as there would be 5 nodeRpcCall functions + 5 watchers + 5 workers to rewrite). I don't want that, as the changes in this PR are quite easy to mess up (forget one `index` -> `id` change or whatever), and doing the rewrite will additionally obscure those.
Lets do this in another PR.

Reply:
Also, it's a big PR already.